src/components/postView/view/postDisplayView.tsx

```diff
@@ -282,6 +284,7 @@ const PostDisplayView = ({
                 <Text style={styles.footerText}>
                   Posted by
```

Comment:
as suggested by feruz, use translations here... there is format supported by intl that supports injecting values in translation entries....

Code: `intl.formatMessage({id:'some_id'}, {username:'some_value_1', appName:'some_value_2'})`
en_US entry:` "some_id":"Posted by $usename via $appName"`